samples/containers/encrypted/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+test_mode=$1
+run_mode=$2
+
+set -e
+
+if [[ "$test_mode" == "clean" ]]; then
+    make clean
+elif [[ "$test_mode" == "init" ]]; then
+    make
+elif [[ "$test_mode" == "run" ]]; then
+    make "$run_mode"-verity
+    # TODO: This doesn't work
+    #make "$run_mode"-integrity
+elif [[ "$test_mode" == "gettimeout" ]]; then
+    # 20 minutes
```

Comment:
wow. out of curiosity, how long does this test usually take?

Reply:
This one doesn't take as long, sorry. The problem was the ``basic/attack`` sample, that can easily take 10-15 minutes in my VM. I just copy pasted the ``test.sh`` script from that folder and forgot to change it.

Reply:
I could be wrong but I think you have 20 minutes for most timeouts that I saw in this PR. I would suggest lowering them if it isn't needed for them.

Reply:
Done.